app/views/partials/_user_post.html.erb

```diff
@@ -0,0 +1,6 @@
+<a href="<%= user_post_path(user, post) %>" class="info-card">
+    <h4 class="user-post-title"><%= post.title %></h4>
+    </br>
+    <p class="user-post-text"><%= post.text %></p>
```

Comment:
- [Optional] Kindly ensure that all the post (not recent_posts) are rendered in the post_index page.